doc/api/n-api.md

```diff
@@ -3879,8 +3880,8 @@ napi_status napi_is_buffer(napi_env env, napi_value value, bool* result)
 
 * `[in] env`: The environment that the API is invoked under.
 * `[in] value`: The JavaScript value to check.
-* `[out] result`: Whether the given `napi_value` represents a `node::Buffer`
-  object.
+* `[out] result`: Whether the given `napi_value` represents a `node::Buffer` or
```

Comment:
`napi_is_typedarray` should be preferred if the caller needs to check if the value is a `Uint8Array`.

Reply:
Committed! Thanks!